src/Bicep.Core/TypeSystem/Az/AzResourceTypeFactory.cs

```diff
@@ -66,6 +67,25 @@ private static TypePropertyFlags GetTypePropertyFlags(Azure.Bicep.Types.Concrete
             return flags;
         }
 
+        private static ObjectType AddResourceProperties(ObjectType objectType)
```

Comment:
Sorry, I should have mentioned - this logic was moved to AzResourceTypeProvider.SetBicepResourceProperties() with the 'existing' PR.

Reply:
Ah, I searched for `AddBicepResourceProperties` in the project when i noticed that merge removed it, and found no matches. Thanks for pointing out the new name, should've scanned through the merge better.